config/set/nette-30.php

```diff
@@ -23,30 +26,47 @@
     // Control class has remove __construct(), e.g. https://github.com/Pixidos/GPWebPay/pull/16/files#diff-fdc8251950f85c5467c63c249df05786
     $services->set(RemoveParentCallWithoutParentRector::class);
 
+    $configuration = [];
+    $configuration[] = inline_service(StaticCallToMethodCall::class)
+        ->args(['Nette\Security\Passwords', 'hash', 'Nette\Security\Passwords', 'hash']);
+    $configuration[] = inline_service(StaticCallToMethodCall::class)
+        ->args(['Nette\Security\Passwords', 'verify', 'Nette\Security\Passwords', 'verify']);
+    $configuration[] = inline_service(StaticCallToMethodCall::class)
+        ->args(['Nette\Security\Passwords', 'needsRehash', 'Nette\Security\Passwords', 'needsRehash']);
```

Comment:
Looking for a better solution actually using the constructor with IDE autocomplete of param names:

```php
$configurations[] = new StaticCallToMethodCall(..., ..., ..., ...);
```


See https://twitter.com/VotrubaT/status/1290016363541454849

Reply:
Best solution so far: https://twitter.com/VotrubaT/status/1290033611987537920